ftp/src/main/scala/akka/stream/alpakka/ftp/impl/SftpOperations.scala

```diff
@@ -155,6 +167,24 @@ private[ftp] trait SftpOperations { _: FtpLike[SSHClient, SftpSettings] =>
     }
   }
 
+  private[this] def authPublickey(identity: SftpIdentity)(implicit ssh: SSHClient) = {
+    def bats(array: Array[Byte]): String = new String(array, "UTF-8")
```

Comment:
```suggestion
    def bats(array: Array[Byte]): String = new String(array, StandardCharsets.UTF_8)
```
I'm not sure if the local method makes the code better.

Reply:
fixed